samples/grpc/shopping-cart-service-scala/k8s/deployment-gke-replicated-dca.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: apps/v1
+kind: Deployment
+# for testing RES over gRPC
+metadata:
+  labels:
+    app: shoppingcartservice
+  name: shopping-cart-service
+  namespace: akka-grpc-projection
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: shoppingcartservice
+  template:
+    metadata:
+      labels:
+        app: shoppingcartservice
+        actorSystemName: ShoppingCartService
+    spec:
+      containers:
+        - name: shopping-cart-service
+          image: gcr.io/akka-team/shopping-cart-service:0.0.0-12-e0d96543-20230109-1410
+          #resources:
+            #limits:
+            #  memory: "2Gi"
+            #requests:
+            #  memory: "2Gi"
+            #  cpu: "3500m"
```

Comment:
I hit some limit in the autopilot kubernetes cluster where it could not allocate machines with that size at all so it failed to deploy. I guess there is some size allocation for the actual k8 nodes somewhere but this was the shortest path.

Reply:
I gave this some more tries but can't get the GKE autopilot to give me more than 500m whatever I do. I tried setting compute class and modifying cpu requests but it just gives me validation error, and I can't find any setting for the kluster itself limiting what kind of resources. For throughput tests we'll have to run on something else than GKE autopilot I guess, treating it as tests verifying that things work for now and not performance tests.